docs/application/native/guides/connectivity/stc.md

```diff
@@ -3,17 +3,17 @@
 
 STC means Smart Traffic Control. It provides extensible packet-level control services, including per-app data usage, total data quota, and background app's data saving. STC Library provides APIs fulfilling below mentioned features for application development.
 
-This feature is supported in mobile and werable profile.
+This feature is supported in mobile, tv and wearable profile.
```

Comment:
Question : Does TV supported by native? I thought for now we have TV only in web profile.

Reply:
Yes, TV profile supports native API for STC F/W.